### ISSUE TITLE
Calculate "days until maturity" for a crop

### ISSUE THREAD

**Issue by Skud** [Growstuff/growstuff#678]
We should calculate how many days from planting (from seed) until harvest for a crop.

We can base this on various data:

- seed.days_until_maturity (mean/median/???) (added in #493)
- actual time from planting to harvest based on GS tracking (requires #501 and might be less accurate)

The former is definitely easier, and should probably be what we use for a first cut of this.

Realistically we're unlikely to be able to do this for at least, say, a few months after #493 is pushed to production.

**Comment by GabrielSandoval** [comment]
Hi @Skud,
I'm an intern at AELOGICA and I'd like to work on this one. I'll be submitting a PR soon.
Thanks.

**Comment by GabrielSandoval** [comment]
/places page:
![screen shot 2015-06-24 at 1 50 09 pm](https://cloud.githubusercontent.com/assets/6015897/8323513/0e406514-1a78-11e5-9584-ecc4e59c9d73.png)

Hi @Skud,
I added this feature and redesigned the plantings index page (see photo above). I added a days_before_planting column on the plantings table. I based the calculations on the previous plantings' start and end date.

Also, I added the days_before_maturity attribute on the show page.
![screen shot 2015-06-24 at 2 01 52 pm](https://cloud.githubusercontent.com/assets/6015897/8323659/05bd2394-1a7a-11e5-84f0-75e39ceb5d5a.png)

Gabriel Sandoval,
AELOGICA Intern
